brave/brave-core
# Fixes date format in contribution_info table

Resolves https://github.com/brave/brave-browser/issues/8528

## Submitter Checklist:

- Verified that these changes build without errors on
- Verified that these changes pass automated tests (unit, browser, security tests) on

## Test Plan:
- go to master or older version
- clean profile with short interval
- enable rewards
- claim grant
- do one time tip, recurring tip and ac
- after they are done open DB and make sure that you have the same problem as in the issue
- switch to this PR
- start the browser
- open DB and make sure that `created_at` is correct now

## Reviewer Checklist:


## After-merge Checklist:

  changes has landed on.
